deploy/operator/setup_hive.sh

```diff
@@ -3,7 +3,37 @@ source ${__dir}/utils.sh
 
 set -xeo pipefail
```

Comment:
Please have no unset as best practice
```suggestion
set -o nounset
set -o pipefail
set -o errexit
set -o xtrace
```